rpctest/memwallet.go

```diff
@@ -481,16 +479,38 @@ func (m *memWallet) CreateTransaction(outputs []*wire.TxOut, feeRate btcutil.Amo
 
 		txIn.SignatureScript = sigScript
 
-		spentOutputs = append(spentOutputs, outPoint)
+		spentOutputs = append(spentOutputs, utxo)
 	}
 
+	// As these outputs are now being spent by this newly created
+	// transaction, mark the outputs are "locked". This action ensures
+	// these outputs won't be doubled spent by any subsequent transactions.
```

Comment:
Fixed.

On Fri, Aug 19, 2016 at 4:29 PM Dave Collins <notifications@github.com>
wrote:

> In rpctest/memwallet.go
> <https://github.com/btcsuite/btcd/pull/371#discussion_r75554172>:
>
> >  	}
> >
> > +	// As these outputs are now being spent by this newly created
> > +	// transaction, mark the outputs are "locked". This action ensures
> > +	// these outputs won't be doubled spent by any subsequent transactions.
>
> s/doubled/double/
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/btcsuite/btcd/pull/371/files/2925fbd6bf40359007399e1bc5295f12f66cf9aa..2a3c3a04f6351c7e2a7e1d719fe77eb5a201ef6f#r75554172>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA87LnLZwnssARtOMirJN7w3ewCHjkXdks5qhiAogaJpZM4D7zxE>
> .
>